docs/connectors/sources/kafka.md

```diff
@@ -88,6 +88,75 @@ requests.put(
 ).raise_for_status()
 ```
 
+### How to write connector config
+
+Below are a couple of examples on how to connect to a Kafka broker
+by specifying
+[the options passed to librdkafka](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md)
+underneath.
+These are simply examples: note that it is strongly recommended outside of test/dev
```

Comment:
"These are simply examples" can be removed.

Reply:
We seem to alternate between connector and adapter

Reply:
Previously, I believe that we decided to prefer "adapter" for the documentation, because otherwise you need to connect a pipeline to a connector and that's a lot of "connect".

Reply:
Good point. But if it's called "connector" in the API, we should probably just stick to that.

Reply:
But it's called "adapter" in the crate :-)

Reply:
I'm fine with any naming scheme that we use consistently -- the API has `/connectors/...` for all its endpoints.

In current pipeline-lingo, a connector is attached to a pipeline.

Reply:
> In current connector-lingo, a connector is attached to a pipeline.

As long as we have distinct vocabulary for the concepts, that's fine. So, no real objection.